app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -34,7 +34,7 @@ def button
   def scan_images
     assert_privileges("image_scan")
     showlist = @lastaction == "show_list"
-    ids = showlist ? find_checked_items : find_current_item(ContainerImage)
```

Comment:
@martinpovolny Yes, you are right. I move RBAC check to ` find_current_item` function.